src/ray/raylet/monitor.h

```diff
@@ -0,0 +1,50 @@
+#ifndef RAY_RAYLET_MONITOR_H
+#define RAY_RAYLET_MONITOR_H
+
+#include <memory>
+#include <unordered_set>
+
+#include "ray/gcs/client.h"
+#include "ray/id.h"
+
+namespace ray {
+
+namespace raylet {
+
+class Monitor {
+ public:
+  // Create a Raylet monitor attached to the given GCS address and port.
```

Comment:
`//` -> `///` for some places in this file, right?

Reply:
Ah sorry, thanks!